template/config/app/app.vue

```diff
@@ -3,28 +3,70 @@
 <template>
   <div id="app" class="bg-gray-400">
     <h1>Vue Starter Project</h1>
-    <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/9/95/Vue.js_Logo_2.svg/1200px-Vue.js_Logo_2.svg.png" class="logo"/>
+    <img
+      src="https://upload.wikimedia.org/wikipedia/commons/thumb/9/95/Vue.js_Logo_2.svg/1200px-Vue.js_Logo_2.svg.png"
+      class="logo"
+    />
     <div class="call-sign">
-      <div class="powered-by">powered<br/>by</div>
-      <div class="tool"><a target="_blank" href="https://www.npmjs.com/package/rdvue">rdvue</a></div>
+      <div class="powered-by">
+        powered
+        <br />by
+      </div>
+      <div class="tool">
+        <a target="_blank" href="https://www.npmjs.com/package/rdvue">rdvue</a>
+      </div>
     </div>
     <div class="contents">
```

Comment:
please update the css styling of this 'contents' element class to have the following styles added
`max-width: 1200px;
 margin: 0 auto;`

Reply:
addressed